src/FixtureInterface.php

```diff
@@ -14,5 +14,5 @@ interface FixtureInterface
     /**
      * Load data fixtures with the passed EntityManager
      */
-    public function load(ObjectManager $manager);
+    public function load(ObjectManager $manager): void;
```

Comment:
This change can impact every project using data-fixtures. Adding the `@return` type annotation will ease migration: #472